plugins/catalog-backend-module-gitlab-org/config.d.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
+
+export interface Config {
+  catalog?: {
+    providers?: {
+      /**
+       * GitlabDiscoveryEntityProvider configuration
+       */
+      gitlab?: {
```

Comment:
Hm. So this is in two places now right? And they are consumed by both plugins and may be at risk of diverging right?

Maybe we can't do a whole lot about that, except perhaps introducing a `@backstage/plugin-catalog-backend-module-gitlab-common` with the config and parsing code, that both modules import?

Reply:
It would be in conflict and the "root" should be changed here.

```suggestion
      gitlabOrg?: {
```

would fit to `GitlabOrgDiscoveryEntityProvider` while `gitlab` is used for `GitlabDiscoveryEntityProvider` from the other module.